backend/typescript/package.json

```diff
@@ -25,6 +25,7 @@
     "lodash": "^4.17.21",
     "mongoose": "^5.11.17",
     "node-fetch": "^2.6.1",
+    "nodemailer": "^6.5.0",
```

Comment:
Do we need the type declarations in `@types/nodemailer`?

Reply:
Yes, for some reason it didn't complain without it but I will add